test/integration/util/util.go

```diff
@@ -81,32 +104,5 @@ func StartScheduler(clientSet clientset.Interface) (factory.Configurator, Shutdo
 		close(stopCh)
 		klog.Infof("destroyed scheduler")
 	}
-	return schedulerConfigurator, shutdownFunc
```

Comment:
@misterikkit How can I return schedulerConfigurator? Need to get the schedulerConfigurator here, otherwise we need to delete a lot of code related to it.